libraries/AP_Motors/AP_MotorsHeli.cpp

```diff
@@ -464,6 +493,22 @@ bool AP_MotorsHeli::parameter_check(bool display_msg) const
         return false;
     }
 
+    // returns false if _collective_min_deg is not default value which indicates users set parameter
+    if (is_equal((float)_collective_min_deg, (float)AP_MOTORS_HELI_COLLECTIVE_MIN_DEG)) {
```

Comment:
I don't think this will work as intented.  One of my helis has min and max collective angles exactly at the defaults, so I will keep getting flagged as not having configured correctly.  A slightly better, but not perfect solution would be to use the method for checking if params have been set from the default value (even if it is the same numeric value).  So it would be better to do:
```suggestion
    if (!_collective_min_deg.configured()) {
```
However, this still relys on uses having actually changed and written the param.  If they happen to leave it at the default they will be left scratching there head as to why they keep getting this warning.  A slightly non-conventional way to address this would be to have more of a non-sense collecive angle default.  e.g. 370 deg.  or zero deg.  We can then at least reliably catch when users have actually configured the param.

Reply:
I thought 90 deg was a non sense value.  That’s why I am using it this way.

Reply:
Ok, thats my mistake.  I miss read what you were doing here.  For some reason I thought min was -2 and max was 10.  Thats my bad.  This looks sensible then.